### PR TITLE
Updated carousel Indigo.Design System section

### PR DESCRIPTION
**Pull request by YoannaIvanova**

### CODE REVIEW

File: en/components/carousel.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ Use the Carousel Component to let the user navigate through a collection of imag
```
Comment (StefanIvanov) [inline en/components/carousel.md:14]:
```suggestion

## Indicators and Slides

The Carousel lets you define up to 10 slides, where for each one there is a matching indicator dot, allowing the user to navigate to that respective slide.

```

File: en/components/carousel.md [hunk 2]
```diff
@@ -14,7 +14,7 @@ Use the Carousel Component to let the user navigate through a collection of imag
 
 ## Styling
```
Comment (StefanIvanov) [inline en/components/carousel.md:15]: @YoannaIvanova add an image for the section above showing a default slider with the 10 dots and some image that you like :)

**Review by StefanIvanov — CHANGES_REQUESTED** [review]

File: en/components/carousel.md (hunk 1 above)
Comment (StefanIvanov) [reply en/components/carousel.md:14]: @YoannaIvanova  add an image for this section showing a default slider with 10 dots and just use some image that you like :)

File: en/components/carousel.md [hunk 3]
```diff
@@ -12,6 +12,10 @@ Use the Carousel Component to let the user navigate through a collection of imag
 
 <img class="responsive-img" src="../images/carousel_demo.png" srcset="../images/carousel_demo@2x.png 2x" />
 
+## Indicators and Slides
+
+The Carousel lets you define up to 10 slides, where for each one there is a matching indicator dot, allowing the user to navigate to that respective slide.
+
```
Comment (StefanIvanov) [inline en/components/carousel.md:18]: I think we need to update all images for this topic except for the dos and donts

**Review by StefanIvanov — CHANGES_REQUESTED** [review]

File: en/components/carousel.md (hunk 3 above)
Comment (StefanIvanov) [reply en/components/carousel.md:18]: And also add an image after this section that is new.

**Review by StefanIvanov — APPROVED** [review]

**Review by sdimchevski — COMMENTED** [review]
Please fix the path for the Retina image here:
scrset="..images/carousel_indicators&slides@2x.png 2x"
should be:
scrset="../images/carousel_indicators&slides@2x.png 2x"

**Comment by YoannaIvanova** [comment]
> Please fix the path for the Retina image here:
> scrset="..images/carousel_indicators&[slides@2x.png](mailto:slides@2x.png) 2x"
> should be:
> scrset="../images/carousel_indicators&[slides@2x.png](mailto:slides@2x.png) 2x"

Fixed

**Review by sdimchevski — APPROVED** [review]

**Pull request merged**